ExtensionExamples/ActionExtension/DocumentAction.cs

```diff
@@ -30,11 +30,19 @@ public class DocumentAction : ActionConnector
             {
                 foreach (var doc in properties.Documents)
                 {
+                    string fileName = properties.JobGuid.ToString() + " - " +
+                        doc.DisplayName + doc.Extension;
+
                     using (var docStream = await properties.GetDocumentStreamAsync(doc))
                     using (var destStream = new MemoryStream())
                     {
                         // Write or send the document to a file, external service, etc
                         await docStream.CopyToAsync(destStream);
+
+                        using (FileStream file = new FileStream(Path.Combine("C:\\temp\\" + fileName), FileMode.Create, FileAccess.Write))
+                        {
+                            destStream.WriteTo(file);
+                        }
```

Comment:
The MemoryStream should be removed if we want to have the sample write to a file. It can go from docStream directly to the file stream